typescript/object_test.go

```diff
@@ -0,0 +1,21 @@
+package typescript
+
+import "testing"
+
+func TestObjectSpreads(t *testing.T) {
+	expected := `
+let myConfig = {
+    ...someOtherObject,
+    a: 45,
+    b: 'test',
```

Comment:
do you need the ability to spread post property assignments? thats common for providing defaults

```ts
    a: 45,
    b: 'test',
    ...someOtherObject,
```

maybe add a second AddSpread method or include a bool to be pre/post properties

Reply:
I don't currently need it, but I'll add it for when i inevitably do